assets/sass/v2/_phone-challenge.scss

```diff
@@ -1,6 +1,10 @@
 .phone-authenticator-challenge {
   .okta-form-subtitle {
     text-align: center;
+
+    .okta-form-subtitle--description {
+      margin-top: 1rem;
+    }
```

Comment:
The one advantage of having Modifier syntax is that this can be written as -
```
&--<modifier> { // With the current text &--description {
  margin-top: 1rem;
}
```
Your current CSS will work as long as the parent-child relationship of `<div class="okta-form-subtitle">` and `<p class="okta-form-subtitle--description">` are maintained. By changing it to above, you'll be targetting `okta-form-subtitle--description` directly and not the relationship.

Reply:
I don't have a strong opinion on this, but leaving it to you. I am ok with either way.